### DISCUSSION TITLE
Status Monitor lacks a way to notify the crew which Periscope to use

### DISCUSSION THREAD

**Discussion by WJohn** [FakeFishGames/Barotrauma#13583] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When piloting the submarine, the Status Monitor is an invaluable tool to report breaches as they become visible on the General Status view, report mechanical issues as they become visible on the Electrical View, and call out other issues should they arise on the radio.

One of the main things any good Captain does is to report issues on the monitor to make it visible to the crew where to go and what to fix. As of writing, there is no way to do this with periscopes, everything is entirely instructed through speech. This is the equivalent of having to say "Breach in ballasts -- the third ballast on the right I mean!", but with guns. The Captain constantly has to go "Enemies above get on the railgun and double coilgun, top left gun top left gun, 10 o'clock!" and, especially depending on the submarine's Periscope layout, may not be immediately obvious which Periscope relates to which gun.

Well, what if Captains could simply click on the gun in the Status Monitor to make a temporary marker on the related Periscope?
Here's a mockup example:
![pic1](https://github.com/FakeFishGames/Barotrauma/assets/5988156/4c015a1f-d1bf-43fe-8455-669f98fa0629)
![pic3](https://github.com/FakeFishGames/Barotrauma/assets/5988156/7c3cbe01-5532-4312-b9ca-782e841f7e6a)
![pic2](https://github.com/FakeFishGames/Barotrauma/assets/5988156/3cc8d171-ddb5-4856-a82c-acfaf1dfcc05)

### Reproduction steps

1. Start a multiplayer campaign.
2. Use the Status Monitor by the Navigation Terminal
3. Mark rooms around the ship with hull breach tags
4. Attempt to place operate weapons tags on the periscopes

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.13.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This is a feature request, not a bug report, moving to discussions.
